docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -47,27 +47,70 @@ represents **all** forms of anemia that are attributable to several causes,
 including causes such as hemoglobinopathies and hemolytic anemias that are not 
 causally attributable to iron deficiency. 
 
-Iron Deficiency Risk Factor
-+++++++++++++++++++++++++++
-
-The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
-hemoglobin concentrations. The risk factor is a **continuous measure**.
-
-.. todo::
-
-	Describe the population and distribution type (normal vs. ensemble)
+Anemia Impairment
++++++++++++++++++
+
+The anemia impairment in GBD 2017 represents the total prevalence of anemia due 
+to all causes modeled in GBD (ex: dietary iron deficiency anemia, anemia due to 
+maternal hemorrhage, sickle cell anemia, etc.). Estimating the total prevalence 
+of the anemia impairment for a given population is the first step in modeling 
+anemia in GBD 2017. This is done by fitting a distribution of hemoglobin levels 
+for that population from primary input data. For GBD 2017, an ensemble 
+distribution was used, which was 40% gamma and 60% mirror gumbel. Source code 
+for this process is available `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia/browse/model/envelope>`_.
+
+Once a the distribution is fit to hemoglobin levels for a particular age-, 
+sex-, and location-specific demographic group, age- and sex-specific hemoglobin 
+thresholds, as defined by the WHO and described in the table below, are used to 
+determine the prevalence of anemia by severity for that population.
 
-Modeling Strategy for the Iron Deficiency Risk Factor
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. list-table:: Hemoglobin Thresholds (g/L)
+	:widths: 15, 15, 15, 15
+	:header-rows: 1
 
-.. todo::
+	* - Group
+	  - Mild Anemia
+	  - Moderate Anemia
+	  - Severe Anemia
+	* - Males and Females <1 month
+	  - 130-149
+	  - 90-129
+	  - <90
+	* - Males and Females 1 month - 5 years
+	  - 100-109
+	  - 70-99
+	  - <70
+	* - Males and Females 5-14 years
+	  - 110-114
+	  - 80-109
+	  - <80
+	* - Males 15+ years
```

Comment:
Your conventions for age ranges are inconsistent here: It appears that "1 month - 5 years" means up to but not including 5 years old, whereas "5 - 14 years" means up to 15 years old, not 14.

If you changed it to "1 month - 4 years", it would match what's in the YLD appendix, or if you changed the other age range to "5 - 15 years", that would be fine too (perhaps preferable).

Or if you wanted to be totally unambiguous, you could instead say "1 month - <5 years" and " 5 - <15 years" or something (but maybe that looks weird).

Reply:
use "demographer notation" for docs, it is what global health people expect.  In this case, "1 month to 4 years".

Reply:
Thanks! It's updated to this notation in the latest commit